cpp-package/example/utils.h

```diff
@@ -42,7 +42,7 @@ bool check_datafiles(const std::vector<std::string> &data_files) {
   return true;
   }
 
-bool setDataIter(MXDataIter *iter , std::string useType,
+bool setDataIter(MXDataIter *iter , const std::string &useType,
               const std::vector<std::string> &data_files, int batch_size) {
```

Comment:
```suggestion
               const std::vector<std::string> &data_files, const int batch_size) {
```
nit: could this be better?

Reply:
no, you don't need to specify "const int" for a "int", as C or C++ passing value, even you changed the batch_size in the function(it basically it is a copy from original one), it doesn't change the original batch_size from the caller. So normally nobody will use const for a int or any primitive type.